docs/sources/static/configuration/agent-management.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Agent Management
+weight: 700
+aliases:
+- ../../configuration/agent-management/
```

Comment:
This is a new file, so you don't need to add the alias. You only need this if the file existed previously but got renamed or moved into a new folder.